containers/Ideas/IdeasList.js

```diff
@@ -5,7 +5,7 @@ import { IdeasListItem } from './IdeasListItem';
 
 export const IdeasList = (props) => {
   //do not pass ideas to child components
-  const {ideas, ...otherProps} = props;
```

Comment:
Haha, this was actually on purpose to remove the ideas from props and then only pass otherProps to child components :) There probably is a better way to do it? But if we remove it, we should then also remove the comment as this is wrong now.